src/main.js

```diff
@@ -0,0 +1,98 @@
+// Use Node-style imports for dependencies.
+const axios = require('axios');
+const readline = require('readline');
+
+// Setup for the readline library.
+const reader = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+// Custom error class to represent a generic failure in the Pets API.
+class PetsApiError extends Error {
```

Comment:
Do students need to implement anything here? Can we one-line `class PetsApiError extends Error { }`?